samples/iothub-connect-to-eventhubs/IotHubToEventHubsSample.csproj

```diff
@@ -10,8 +10,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Azure.Messaging.EventHubs" Version="5.6.0" />
-    <PackageReference Include="Microsoft.Azure.Amqp" Version="2.5.3" />
+    <PackageReference Include="Azure.Messaging.EventHubs" Version="5.7.2" />
+    <PackageReference Include="Microsoft.Azure.Amqp" Version="2.5.10" />
```

Comment:
Should we just remove this dependency and allow it to be pulled in via Event Hubs?

Reply:
We use it directly in the sample; I didn't want to pull it in just via transitive dependency - it felt confusing.   Thoughts?

Reply:
Ah that makes sense - didn't realize it was used in the sample.